templates/cloud/openstack/autopilot.html

```diff
@@ -55,100 +55,93 @@
     <h2>Build an OpenStack cloud with Autopilot</h2>
 
     <div class="six-col">
-        <h3>Install Autopilot on your own servers</h3>
         <p><b>Experience OpenStack at its best</b></p>
         <p>OpenStack Autopilot is the world’s easiest dynamic OpenStack installer. It handles every aspect of your cloud both during installation, expansion, and everyday operations.</p>
-            <p>Requirements for Autopilot:</p>
-            <ul class="list-ticks--compact">
-                <li>At least five machines with two disks</li>
-                <li>Two machines with x2 network interfaces (NICs)</li>
-                <li>A dedicated switch to create a private cloud LAN</li>
-                <li>Internet access through a router on that LAN</li>
-            </ul>
-            <a class="button--primary" href="/download/cloud/install-openstack-with-autopilot">
-                Get Autopilot for your hardware
+        <p>
+            <a class="button--primary" href="/download/cloud">
+            Get Autopilot
             </a>
+        </p>
+    </div>
+
+    <div class="six-col last-col">
+        <p><b>Requirements for Autopilot:</b></p>
```

Comment:
I know that you didn't introduce this but could we replace these with something nicer. <h4> looks good and is better, semantically, for a header